build-tools/android-toolchain/android-toolchain.projitems

```diff
@@ -30,11 +30,11 @@
       <HostOS>Darwin</HostOS>
       <DestDir>build-tools\$(XABuildToolsVersion)</DestDir>
     </AndroidSdkItem>
-    <AndroidSdkItem Include="platform-tools_r23-macosx.zip">
+    <AndroidSdkItem Include="platform-tools_r26.0.0-rc1-macosx.zip">
       <HostOS>Darwin</HostOS>
       <DestDir>platform-tools</DestDir>
     </AndroidSdkItem>
-    <AndroidSdkItem Include="tools_r24.4.1-macosx.zip">
+    <AndroidSdkItem Include="tools_r25.3.1-macosx.zip">
```

Comment:
`build-tools_r26.0.0-rc1-macosx.zip` doesn't exist [causing a failure](https://jenkins.mono-project.com/job/xamarin-android-pr-builder/738/):

    Unable to download URL `https://dl-ssl.google.com/android/repository/tools_r25.3.1-macosx.zip` to `/Users/builder/android-archives/tools_r25.3.1-macosx.zip`: 404 (Not Found)